{"uid":"ad6cdd5a1d86abdc","name":"test_upgrade_from_edge","fullName":"tests.integration.high_availability.test_upgrade#test_upgrade_from_edge","historyId":"68531e26becef799dab824000e21a3c5","time":{"start":1756777963829,"stop":1756777965925,"duration":2096},"status":"broken","statusMessage":"mysql.connector.errors.ProgrammingError: 1049 (42000): Unknown database 'continuous_writes_database'","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Unknown database 'continuous_writes_database'\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/high_availability/test_upgrade.py\", line 100, in test_upgrade_from_edge\n    await ensure_all_units_continuous_writes_incrementing(ops_test, credentials=credentials)\n  File \"/root/spread_project/tests/integration/high_availability/high_availability_helpers.py\", line 488, in ensure_all_units_continuous_writes_incrementing\n    last_max_written_value = await get_max_written_value_in_database(\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/high_availability/high_availability_helpers.py\", line 61, in get_max_written_value_in_database\n    output = execute_queries_on_unit(\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/helpers.py\", line 194, in execute_queries_on_unit\n    cursor.execute(query)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 357, in execute\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.ProgrammingError: 1049 (42000): Unknown database 'continuous_writes_database'","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1756777779898,"stop":1756777779898,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1756777779898,"stop":1756777779898,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1756777779899,"stop":1756777780146,"duration":247},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1756777959743,"stop":1756777959743,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1756777959744,"stop":1756777961790,"duration":2046},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1756777959744,"stop":1756777959744,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"credentials","time":{"start":1756777961790,"stop":1756777963829,"duration":2039},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"mysql.connector.errors.ProgrammingError: 1049 (42000): Unknown database 'continuous_writes_database'","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 706, in cmd_query\n    self._cmysql.query(\n_mysql_connector.MySQLInterfaceError: Unknown database 'continuous_writes_database'\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/high_availability/test_upgrade.py\", line 100, in test_upgrade_from_edge\n    await ensure_all_units_continuous_writes_incrementing(ops_test, credentials=credentials)\n  File \"/root/spread_project/tests/integration/high_availability/high_availability_helpers.py\", line 488, in ensure_all_units_continuous_writes_incrementing\n    last_max_written_value = await get_max_written_value_in_database(\n                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/high_availability/high_availability_helpers.py\", line 61, in get_max_written_value_in_database\n    output = execute_queries_on_unit(\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/helpers.py\", line 194, in execute_queries_on_unit\n    cursor.execute(query)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/cursor_cext.py\", line 357, in execute\n    result = self._connection.cmd_query(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py\", line 97, in wrapper\n    return method(cnx, *args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/mysql/connector/connection_cext.py\", line 714, in cmd_query\n    raise get_mysql_exception(\nmysql.connector.errors.ProgrammingError: 1049 (42000): Unknown database 'continuous_writes_database'","steps":[],"attachments":[{"uid":"95b840e0738892a8","name":"log","source":"95b840e0738892a8.txt","type":"text/plain","size":497}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1756777965973,"stop":1756777966995,"duration":1022},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"credentials::finalizer","time":{"start":1756777965973,"stop":1756777965973,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1756777966996,"stop":1756777966996,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1756777967000,"stop":1756777967431,"duration":431},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1756777967432,"stop":1756777967432,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1756777967432,"stop":1756777967432,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1756777967432,"stop":1756777967432,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.high_availability"},{"name":"suite","value":"test_upgrade"},{"name":"host","value":"pkrvmccyg1gnepe"},{"name":"thread","value":"18559-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.high_availability.test_upgrade"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d0d91eae9b9f0ff1","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":16,"skipped":36,"passed":478,"unknown":17,"total":547},"items":[{"uid":"bbd782ace52cbbdc","reportUrl":"../3193//#testresult/bbd782ace52cbbdc","status":"passed","time":{"start":1756690719366,"stop":1756690911644,"duration":192278}},{"uid":"73c8287b358c9eb9","reportUrl":"../3192//#testresult/73c8287b358c9eb9","status":"passed","time":{"start":1756604091299,"stop":1756604286614,"duration":195315}},{"uid":"4964579e89def17a","reportUrl":"../3190//#testresult/4964579e89def17a","status":"passed","time":{"start":1756519597498,"stop":1756519783379,"duration":185881}},{"uid":"8a0acd8d05d463db","reportUrl":"../3189//#testresult/8a0acd8d05d463db","status":"passed","time":{"start":1756435551412,"stop":1756435741269,"duration":189857}},{"uid":"8c6d15632e18c050","reportUrl":"../3188//#testresult/8c6d15632e18c050","status":"unknown","time":{}},{"uid":"d6aed8fa4d7d0ab7","reportUrl":"../3184//#testresult/d6aed8fa4d7d0ab7","status":"passed","time":{"start":1756259704538,"stop":1756259894425,"duration":189887}},{"uid":"e9b2bab93ec16469","reportUrl":"../3175//#testresult/e9b2bab93ec16469","status":"passed","time":{"start":1756175857150,"stop":1756176070127,"duration":212977}},{"uid":"980598d758258959","reportUrl":"../3171//#testresult/980598d758258959","status":"unknown","time":{}},{"uid":"bab09b01217a0d87","reportUrl":"../3170//#testresult/bab09b01217a0d87","status":"unknown","time":{}},{"uid":"6f178bf72591816c","reportUrl":"../3169//#testresult/6f178bf72591816c","status":"passed","time":{"start":1755914036257,"stop":1755914250651,"duration":214394}},{"uid":"77f563a809fc7c14","reportUrl":"../3166//#testresult/77f563a809fc7c14","status":"passed","time":{"start":1755826832843,"stop":1755827033504,"duration":200661}},{"uid":"b9f8c02ab410984a","reportUrl":"../3161//#testresult/b9f8c02ab410984a","status":"passed","time":{"start":1755741698412,"stop":1755741900057,"duration":201645}},{"uid":"b34669195909ba0","reportUrl":"../3152//#testresult/b34669195909ba0","status":"passed","time":{"start":1755655034839,"stop":1755655248145,"duration":213306}},{"uid":"d6b6839a1a9368ce","reportUrl":"../3143//#testresult/d6b6839a1a9368ce","status":"passed","time":{"start":1755567833075,"stop":1755568019561,"duration":186486}},{"uid":"3014761a5d9d2078","reportUrl":"../3142//#testresult/3014761a5d9d2078","status":"passed","time":{"start":1755481207040,"stop":1755481424299,"duration":217259}},{"uid":"dc344f1ab9bd948a","reportUrl":"../3140//#testresult/dc344f1ab9bd948a","status":"passed","time":{"start":1755309172975,"stop":1755309362780,"duration":189805}},{"uid":"8b671777ffd24c66","reportUrl":"../3139//#testresult/8b671777ffd24c66","status":"passed","time":{"start":1755221583851,"stop":1755221796158,"duration":212307}},{"uid":"a9273183e076383d","reportUrl":"../3138//#testresult/a9273183e076383d","status":"passed","time":{"start":1755136887951,"stop":1755137097392,"duration":209441}},{"uid":"c057f819f050ad95","reportUrl":"../3137//#testresult/c057f819f050ad95","status":"passed","time":{"start":1755051397437,"stop":1755051607256,"duration":209819}},{"uid":"d0530f66807a46e0","reportUrl":"../3136//#testresult/d0530f66807a46e0","status":"passed","time":{"start":1754962847489,"stop":1754963033801,"duration":186312}}]},"tags":["asyncio","abort_on_fail"]},"source":"ad6cdd5a1d86abdc.json","parameterValues":[]}